packages/webapp/next.config.mjs

```diff
@@ -1,7 +1,28 @@
+const {
+  VERCEL_ENV = 'development',
+  VERCEL_URL = 'localhost'
+} = process.env
```

Comment:
should probably be included in .env.example

Reply:
Agreed. Though I'd say we're at the point where we need to break out specific localhost dev vs anything on Vercel vars in  `.env.example` and `env.local.example`

Reply:
I know this is already merged, but as a correction to this @0xMillz. You are right. I believe I put in a different comment response what you said here, that those env vars are already on `process.env.{AUTO_GENERATED_ENV_VAR}`.